actions/submit-signing-request/utils.ts

```diff
@@ -1,4 +1,6 @@
 import * as moment from 'moment';
+import * as core from '@actions/core';
+import { AxiosError, AxiosResponse } from 'axios';
 
 /// function that retries promise calls with delays
 /// the delays are incremental and are calculated as follows:
```

Comment:
Unrelated to this PR: I just noticed that the log output is wrong in `executeWithRetries`
* a) you're using console.log instead of the Github Actions logging system (maybe that's okay?)
* b) the log message needs to be two lines below, *after* `delayMs` is recalculated, otherwise it is just wrong

Reply:
a) - Console logs also end up in the task log. However, you are right – it's better to use core abstraction.

b) - The log message informs how long the next waiting interval will be. In the next line, we have the waiting code. So, I think it's the right place. Or maybe I'm missing something; let's discuss.